src/services/Query.ts

```diff
@@ -0,0 +1,15 @@
+import { ApiPromise } from '@polkadot/api'
+import { AccountInfo } from '@polkadot/types/interfaces'
+
+type Iquery = {
+  getCurrentNonce(api: ApiPromise, address: string): Promise<string>
+}
+
+const getCurrentNonce = async (api: ApiPromise, address: string): Promise<string> => {
```

Comment:
It is worth to expose `getNonce` in Mangata class also

Reply:
sure I will expose getNonce in Mangata class